pkg/pluginmanager/manager.go

```diff
@@ -825,24 +825,43 @@ func InstallPluginsFromGroup(pluginName, groupID string) error {
 
 	numErrors := 0
 	numInstalled := 0
+	mandatoryPluginsExist := false
+	pluginExist := false
 	for _, plugin := range group.Plugins {
-		if (pluginName == cli.AllPlugins || pluginName == plugin.Name) && plugin.Mandatory {
-			err = InstallStandalonePlugin(plugin.Name, plugin.Version, plugin.Target)
-			if err != nil {
-				numErrors++
-				log.Warningf("unable to install plugin '%s': %v", plugin.Name, err.Error())
-			} else {
-				numInstalled++
+		if pluginName == cli.AllPlugins || pluginName == plugin.Name {
+			pluginExist = true
+			if plugin.Mandatory {
+				mandatoryPluginsExist = true
+				err := InstallStandalonePlugin(plugin.Name, plugin.Version, plugin.Target)
+				if err != nil {
+					numErrors++
+					fmt.Printf("unable to install plugin '%s': %v", plugin.Name, err.Error())
```

Comment:
@mpanchajanya Any reason you changed this from `log.Warningf` to `fmt.Printf`?

The former prints to stderr while the latter prints to stdout so when getting an error, the output is not on the same stream.

Also, `log.Warningf` seems to automatically add a newline at the end, which is important in this case.

I can make a quick PR to fix it but I wanted to know if there was a reason behind it before messing it up.

Reply:
Fixed in #226